package.json

```diff
@@ -87,6 +87,7 @@
     "q": "1.0.1",
     "qrcode.react": "0.6.1",
     "qs": "4.0.0",
+    "ramda": "0.22.1",
```

Comment:
> added bloat 

This didn't seem like a problem to me since we're removing hundreds of kilobytes by taking out the iframe'd code. I realize there's confusion, which is something to consider, but with that in mind, the notifications code has always been different anyway and it's tightly wound up in the code.

**lodash**'s moves towards a functional API in `lodash/fp` was a major let-down and I found it hard to work with. it's something here @aduth that I can't imagine taking out before getting the new notes client in if we end up doing that. if we don't want it, I think it would be better to pull it out in the future or simply ban new uses in code reviews

I started using **lodash** when I build this months ago, but found it too annoying, and when I replaced it with **ramda** the build size shrank dramatically

thanks for your comment, though. I don't mean to sound put-offish by dismissing it, but it's just something I think is low priority until we're feature-complete.